.ci/travis-dependencies.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
-
-if [[ $TRAVIS_OS_NAME == "osx" ]] ; then
-  brew update
```

Comment:
Why remove the OSX brew instructions?

Reply:
I see, moved to travis config

Reply:
I think the idea with the external script was to keep the config file itself slimmer and cleaner. Ctrl introduced it back then.